README.md

```diff
@@ -39,6 +39,39 @@ to get started is with the official Docker image, sensu/sensu.
 
 See the [installation documentation](https://docs.sensu.io/sensu-go/latest/installation/install-sensu/) to get started.
 
+### Building from source
+
+The various components of Sensu Go can be manually built from this repository.
+You will first need [Go](https://golang.org/doc/install#install) installed.
+Then, you should clone this repository **outside** of the GOPATH since Sensu Go
+uses [Go Modules](https://github.com/golang/go/wiki/Modules).
+
+To compile and then run Sensu Go within a single step:
+```
+$ go run ./cmd/sensu-agent/main.go
+$ go run ./cmd/sensu-backend/main.go
+$ go run ./cmd/sensuctl/main.go
```

Comment:
Can we also add a note that explains the minimum Golang version required, and these steps (for verbosity): 

```
$ go get github.com/sensu/sensu-go
$ cd $GOPATH/src/github.com/sensu/sensu-go
```

Reply:
We actually recommend now to not use the GOPATH but yeah, adding those steps for verbosity is definitely a good idea, I’ll adapt those!